movies/src/components/NewMovieInput.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+// import {  } from '../actions';
+
+class NewMovieInput extends Component {
+  constructor(props) {
```

Comment:
No need to pass `props` here to `constructor` and `super` since you aren't using `props` inside the constructor.